src/Temporalio/Worker/Interceptors/DelayAsyncInput.cs

```diff
@@ -8,11 +9,23 @@ namespace Temporalio.Worker.Interceptors
     /// </summary>
     /// <param name="Delay">Delay duration.</param>
     /// <param name="CancellationToken">Optional cancellation token.</param>
+    /// <param name="Summary">Summary for the delay.</param>
```

Comment:
Maybe worth saying "which will appear in the workflow history" or something

For new users it might be really nonobvious why a "delay" would have a summary.

Reply:
This the interceptor field (mostly because .NET linting makes me document it). For the one the user calls, in `DelayOptions`, it states:

> Gets or sets a simple string identifying this timer that may be visible in UI/CLI. While it can be normal text, it is best to treat as a timer ID.